extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -112,4 +113,33 @@ public static function onArticleContentOnDiff( $diffEngine, \OutputPage $output
 
 		return true;
 	}
+
+	public static function onRawPageViewBeforeOutput( \RawAction &$rawAction, &$text ) {
+		global $wgCityId;
+
+		$title = $rawAction->getTitle();
+
+		if ( $title->getNamespace() === NS_MEDIAWIKI && $title->isJsPage() ) {
```

Comment:
Preferred method of checking namespaces in MW is `$title->inNamespace( NS_MEDIAWIKI )`.